### PR TITLE
Add configuration field for alerting days

### PR DESCRIPTION
**Pull request by MartinWeindel**
**How to categorize this PR?**

/area control-plane
/kind enhancement

**What this PR does / why we need it**:
Add configuration field for alerting days

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
follow-up of #174 

**Release note**:

```other user
Add configuration field `certExpirationAlertDays` to allow overriding default value in shoot manifest.
```

### CODE REVIEW

File: charts/internal/shoot-cert-management-seed/templates/configmap-observability.yaml [hunk 1]
```diff
@@ -44,8 +44,8 @@ data:
             type: seed
             visibility: operator
           annotations:
-            description: Certificate in namespace {{ .Release.Namespace }} will be expired less than {{ .Values.configuration.certExpirationAlertDays }} days.
-            summary: SSL certificate will expire less than {{ .Values.configuration.certExpirationAlertDays }} days
+            description: Certificate in namespace {{ .Release.Namespace }} will be expired in less than {{ .Values.configuration.certExpirationAlertDays }} days.
+            summary: SSL certificate will expire in less than {{ .Values.configuration.certExpirationAlertDays }} days
```
Comment (ScheererJ) [inline charts/internal/shoot-cert-management-seed/templates/configmap-observability.yaml:48]: Nit.

```suggestion
            description: Certificate in namespace {{ .Release.Namespace }} will expire in less than {{ .Values.configuration.certExpirationAlertDays }} days.
            summary: TLS certificate will expire in less than {{ .Values.configuration.certExpirationAlertDays }} days
```

File: docs/usage/alerting.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+scope: operator
+publishdate: 2023-07-20
+tags: ["task"]
+description: How to change the alerting on expiring certificates
+---
+
+# Changing alerting settings
+
+Certificates are normally renewed automatically 30 days before they expire.
+As a second defense line, there is an alerting in Prometheus activated if the certificate is a few days
```
Comment (ScheererJ) [inline docs/usage/alerting.md:14]:
```suggestion
As a second line of defense, there is an alerting in Prometheus activated if the certificate is a few days
```

File: docs/usage/alerting.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+before expiration. By default, the alert is triggered 15 days before expiration.
+
+You can configure the days in the `providerConfig` of the extension.
+
+In this example, the days are changed to 3 days before expiration.
+
```
Comment (ScheererJ) [inline docs/usage/alerting.md:20]: Can this alert also be disabled, e.g. by setting it to `0` or a negative value? Do we want to support disabling this alert?

**Review by ScheererJ — COMMENTED** [review]
Thanks. Looks good already.

I just have a minor nit and a question.

File: docs/usage/alerting.md (hunk 3 above)
Comment (MartinWeindel) [reply docs/usage/alerting.md:21]: Added disabling alert by setting it to `0`.

Comment (ScheererJ) [reply docs/usage/alerting.md:21]: Thanks.

/lgtm

**Review by ScheererJ — APPROVED** [review]

**Pull request merged**